Makefile

```diff
@@ -533,11 +533,14 @@ run_bpf_tests: ## Build and run the BPF unit tests using the cilium-builder cont
 run-builder: ## Drop into a shell inside a container running the cilium-builder image.
 	DOCKER_ARGS=-it contrib/scripts/builder.sh bash
 
+## Run a local linter for the renovate configuration
```

Comment:
Moving this comment above the target breaks the help text FYI.

`main`:
```
$ make help | grep renovate
  renovate-local                Run a local linter for the renovate configuration
$
```

This change:
```
$ make help | grep renovate
$
```